als-suggestions/shared/src/test/resources/test/oas20/by-directory/json/responses/expected/response-header.json.json

```diff
@@ -209,6 +209,66 @@
     "commitCharacters": null,
     "command": null
   },
+  {
+    "label": "items",
```

Comment:
are we suggesting items/properties and other specific entries on purpose? this would be invalid if there is no `type: array` for example

Reply:
Seems to be on purpose, see `def defaultSuggestions` on `WebApiTypeFacetsCompletionPlugin`